packages/suite/src/views/wallet/sign-verify/components/HiddenAddressRow.tsx

```diff
@@ -58,12 +37,18 @@ interface HiddenAddressRowProps {
 export const HiddenAddressRow = ({ item, variant, className }: HiddenAddressRowProps) => {
     const { elevation } = useElevation();
 
+    const map: Record<OverlayVariant, Elevation> = {
+        option: elevation,
+        'option-focused': nextElevation[elevation],
+        input: elevation,
+    };
+
```

Comment:
`map` is a quite generic name, IMO. But also this whole thing can be omitted, since `elevation` can be used as a default value and the variant prop can be changed from `variant` to something like `isElevated`. Which means you could simply do:

```
const currentElevation =  isElevated ? nextElevation[elevation] : elevation;

 <StyledGradientOverlay forcedElevation={currentElevation} hiddenFrom="160px" />
```

Reply:
Just for record: I dont mind the `map` naming, in this super local context, the name can be generic as it is used in super-limited context. 

However doing just the `isElevated` prop makes it simpler.